File: C++/Beginner/crescente_decrescente.cpp
// Caio Cesar 01/12/23
// BEE 1113

#include <iostream>
using namespace std;

int main(){
    int num_x, num_y;
    do{
        cin >> num_x >> num_y;

        if(num_x > num_y){
            cout << "Decrescente" << endl;
        }else if(num_x < num_y){
            cout << "Crescente" << endl;
        }

    } while(num_x != num_y);

    return 0;
}

File: Python/Beginners/crescente_decrescente.py
# Caio Cesar 01/12/23
# BEE 1113

while True:
    numeros  = input().split()
    num_x = int(numeros[0])
    num_y = int(numeros[1])

    if num_x > num_y:
        print("Decrescente")
    elif num_x < num_y:
        print("Crescente")
    else:
        break